### ISSUE TITLE
[BUG]: Devil May Cry 3 Special Edition – wrong deinterlace in PAL videomode

### ISSUE THREAD

**Issue by mirodno** [PCSX2/pcsx2#9845]
### Describe the Bug

Title. Most noticeable with weave and adaptive, but bob and blend also seems to work wrong (more artefacts/quality drops then NTSC counterpart)

### Reproduction Steps

Boot up the game, choose PAL videomode, watch up letters and menus. Gameplay isn't any better, compared to NTSC.

### Expected Behavior

PAL videomode is supposed to look clearer, actually. Good example is Killzone, where PAL videomode provides clearer visual clarity (with lower framerate), compared to NTSC.

### PCSX2 Revision

v1.7.4943

### Operating System

Windows 11

### CPU

Ryzen 5 2600

### GPU

RX 550

### GS Window Screenshots

PAL:
![Untitled](https://github.com/PCSX2/pcsx2/assets/41358915/7567cc41-b96c-4fd7-bd3f-ef9fc059f100)
![Untitled2](https://github.com/PCSX2/pcsx2/assets/41358915/8546f41a-c23b-4d00-9058-b8046bbc807c)

NTSC:
![Untitled3](https://github.com/PCSX2/pcsx2/assets/41358915/20d904d9-4646-4074-a75b-68d23f436f87)
![Untitled4](https://github.com/PCSX2/pcsx2/assets/41358915/a6d2448d-f1a1-4279-bf0b-6d0c188dd363)

**Comment by Parallax71** [comment]
I just tried the DMC 3 special edition and what you say doesn't happen to me at 50hz (even less at 60hz as the game runs fine), although what you say is wrong with you, it did happen to me with Dark Cloud, but I solved it by going to "Properties" of the game and there in the "Graphics" section I selected what I send you in the capture, try and tell us if it is solved by doing that.
![solc](https://github.com/PCSX2/pcsx2/assets/142044650/40d421c5-e35d-4462-89af-c3d78835702f)

**Comment by Neutroni** [comment]
Can confirm to happening to me, PCSX2 Nightly - v1.7.5013 on Arch Linux with all default settings, changing deinterlacing does not change anything. Seems to happen with all renderers.
GPU: AMD Radeon RX 480 Graphics

**Comment by refractionpcsx2** [comment]
Looks like it's some internal rounding shinanigans, I checked it with Preround #6553 and got some interesting results.

I disabled Bilinear filtering for this check

Normal:
![image](https://github.com/PCSX2/pcsx2/assets/6278726/02468070-3197-4c44-90c7-6dfed5c494e0)

Prerounded:
![image](https://github.com/PCSX2/pcsx2/assets/6278726/e194cb78-62d9-4114-916f-77d8bfcb7a5f)

So it's still not right, but that could be because the PR is wrong :P but it is interesting that it has an effect

**Comment by refractionpcsx2** [comment]
Just for reference, this is it running on the console

![Screenshot 2023-09-17 09-21-45](https://github.com/PCSX2/pcsx2/assets/6278726/8007cc74-faaf-40a5-b386-d6996683b608)

**Comment by Parallax71** [comment]
It's strange, the text is shown to me completely clear, (I leave a screenshot), I play at 60hz

![Devil May Cry 3 - Dante's Awakening  Special Edition _SLES-54186_20230917113444](https://github.com/PCSX2/pcsx2/assets/142044650/a74c1f40-109c-49cb-8b04-df93da548766)

**Comment by refractionpcsx2** [comment]
Honestly, I'm half tempted to say "replicated on real hardware", as it looks pretty crap on the real machine.

@Parallax71 are you using a texture pack? because that doesn't look like normal text.

**Comment by Parallax71** [comment]
> are you using a texture pack? because that doesn't look like normal text.

No, I use what is by default, I have not modified anything.

**Comment by refractionpcsx2** [comment]
oh also note, you can't see this when upscaling, it's native where this appears.

**Comment by Parallax71** [comment]
> oh also note, you can't see this when upscaling (though another bug can happen), it's native where this appears.

the language, it is because of the language, put in English it appears like this to me...

![Devil May Cry 3 - Dante's Awakening  S  E _SLES-54186_20230917114250](https://github.com/PCSX2/pcsx2/assets/142044650/7d4a2ffd-6d0c-4a4c-9837-1489e6ea5fda)

HW mode at 2x (720p) - Vulkan

**Comment by refractionpcsx2** [comment]
how strange, they must have done the font glyphs differently, or something.

As I said, upscaling (2x specifically) looks fine, you need to be in native resolution, or something quite high

**Comment by Parallax71** [comment]
> how strange, they must have done the font glyphs differently, or something.
> 
> As I said, upscaling (2x specifically) looks fine, you need to be in native resolution, or something quite high

Yes, in Spanish they changed the typography, you can see that it is different, and using the 2x or 3x resolution helps a lot to eliminate that problem.

**Comment by mirodno** [comment]
Oh wow, I didn't know the problem was in the original, or maybe interlacing on my TV was too good back then :^) 

Perhaps the issue should be closed, there is no point in fixing something that is “not broken” by emulation terms (better to leave this matter to patches). But I'll leave this decision to developers @refractionpcsx2 

Speaking of patches, at my guess, the problem may be because of that both versions use the same "offset" (or whatever it is, I don’t know how to call it, so I'll use this term for simplicity sake) for NTSC and PAL. Is it possible to hack these "offsets" or whatever it is? It might also help with the other games that share similar problem.
